CONTRIBUTING.rst

```diff
@@ -35,9 +35,9 @@ Tests
 There are not a huge number of tests right now, as most of the code in Prospector is
 handling the output of other tools. However, please do run them before submitting a pull request::
 
-    nosetests tests/
+    pytest
 
-Prospector targets Python 3.6, 3.7, 3.8 and 3.9. You can use `tox`_ to test this locally,
+Prospector targets Python 3.7, 3.8, 3.9, and 3.10. You can use `tox`_ to test this locally,
```

Comment:
We're still targeting 3.6, for now, see https://github.com/PyCQA/prospector/pull/511#issuecomment-1156178453

Reply:
My guess is that you and I are the only ones that will read this file between not and the dropping of py36 and this pull request demonstrates that this file is updated as an afterthought.
https://devguide.python.org/versions/#versions

Reply:
I wanted to wait for 511 to be merged to merge this one just after (and keep your original change) considering the cache issue. But let's merge if the cache issue is fixed. It could be due to a new interpreter being released so the cache is confused about it.